packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -517,6 +523,7 @@ const UnifiedDesignPicker: React.FC< UnifiedDesignPickerProps > = ( {
 				onPreview={ onPreview }
 				onUpgrade={ onUpgrade }
 				designs={ staticDesigns }
+				blankCanvas={ blankCanvas }
```

Comment:
I think the prop name `blankCanvas` is confusing here because we are already using the variable `isBlankCanvas` in the same file. 

We could use the prop name `buildYourOwnCta`, which would make more sense because we use the name **Build your own** in the step and the flag name.